example/Gemfile

```diff
@@ -3,4 +3,10 @@ source 'https://rubygems.org'
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby '>= 2.6.10'
 
-gem 'cocoapods', '>= 1.12.0'
+# Exclude problematic versions of cocoapods and activesupport that causes build failures.
+gem 'cocoapods', '>= 1.13', '!= 1.15.0', '!= 1.15.1'
+gem 'activesupport', '>= 6.1.7.5', '!= 7.1.0'
+gem 'xcodeproj', '< 1.26.0'
+
+# Fix vulnerabilities
+gem 'rexml', '>= 3.3.9'
```

Comment:
Enforce version of the `rexml` transitive dependency to fix the vulnerability in `Gemfile.lock` files